recipes/pangolin/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.0.2" %}
+
+package:
+  name: pangolin
+  version: {{ version }}
+
+source:
+  - url: https://github.com/hCoV-2019/pangolin/archive/{{ version }}.tar.gz
+    sha256: f171f9b54a0e6f8b18135a50c484c9b1923cb1b2f3f93f445671a18ed212fbf6
+    folder: pangolin
+  - url: https://github.com/hCoV-2019/lineages/archive/2020-04-27.tar.gz
+    sha256: 0a998177c5395386e4d18a215c37db7a9af03d55f110ed319c94810ff7f86522
+    folder: lineages
+
+
+build:
+  number: 0
+  noarch: generic
+  script: {{ PYTHON }} -m pip install ./pangolin/. --no-deps -vv && {{ PYTHON }} -m pip install ./lineages/. --no-deps -vv
+
+requirements:
+  host:
+    - python =3.6
+    - pip =19.3.1
```

Comment:
Are newer versions of python really not supported? Do you really need this exact version of pip? As a general rule, it's best to avoid specify exact version requirements when possible.

Reply:
I was following their specified versions in their file instructions. I'll attempt to do some changes and see if the build still works

Reply:
Changed the host ones, but I didn't play around too much with the other ones. Should I try to see if they all work without exact versions?

Reply:
Should be better